Minor changes - based on suggestions made by F5 S6 (#482)
withastro/docs: src/pages/en/core-concepts/astro-pages.md

## Non-HTML Pages

Non-HTML pages, like `.json` or `.xml`, or even assets such as images, can be built using API routes commonly referred to as **File Routes**. 

> **File Routes** are script files which end with the `.js` or `.ts` extension, eith the source file must exist within the `src/pages/` directory.

Reviewer: ```suggestion
**File Routes** are script files that end with the `.js` or `.ts` extension and are located within the `src/pages/` directory.
```
What about this?
